### PR TITLE
update command summary for updatecommand

### PR DESCRIPTION
**Pull request by dvdweien**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -377,8 +377,9 @@ Action | Format, Examples
 **List all** | `list all`
 **List property** | `list property`
 **List appointment** | `list appointment`
-**Add new status** | `update INDEX new AMOUNT`<br><br> e.g.,`update 1 new 600000`
-**Update status** | `update INDEX u/STATUS`<br><br> e.g. `update 3 u/proceed`
+**Add new status** | `update INDEX u/new AMOUNT`<br><br> e.g.,`update 1 u/new 600000`
+**Proceed to the next status** | `update INDEX u/proceed`
+**Remove a status** | `update INDEX u/cancel`
```
Comment (w2vgd) [inline docs/UserGuide.md]: Just wondering if you separate out this 3 formats of update in the command summary, then should you separate them in the Command section above too? Its better to be consistent so if you want to separate out to 3 formats, then both sides should be separated. If not, both sides should be 1 format only.

**Review by w2vgd — COMMENTED** [review]
A suggestion

File: docs/UserGuide.md (hunk 1 above)
Comment (dvdweien) [reply docs/UserGuide.md]: ok! changed it!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -216,15 +216,15 @@ Updates the status of a property from Option to Purchase, to Sales and Purchase
 
 Formats:
 * `update INDEX u/new AMOUNT`
-* `update INDEX u/[proceed][cancel]`
+* `update INDEX u/proceed`
+* `update INDEX u/cancel`
```
Comment (w2vgd) [inline docs/UserGuide.md]: Actually will it be better if you have only 1 format, `update INDEX u/OPTION`, then you specify what the options are? Similar to how `find` is structured?

Not really sure whats the best way to present this...

File: docs/UserGuide.md [hunk 3]
```diff
@@ -216,15 +216,15 @@ Updates the status of a property from Option to Purchase, to Sales and Purchase
...
 
 Description:
-* Edits the property at the specified `INDEX`. The index refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​
+* Updates the status of the property at the specified `INDEX`. The index refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​
 * The `new` keyword can only be used on a property without an existing status
 * `proceed` or `cancel` can only be used on a property with an existing status
 * `proceed` would move the status on to the next one. e.g. Option to Sales Agreement or Sales Agreement to Completion
 * `cancel` would remove the status of the property
-* At least one of the optional fields must be provided.
```
Comment (w2vgd) [inline docs/UserGuide.md]: Should you also add a bit more information on what each of the status represents, and calling proceed on one will lead to which one?

**Review by w2vgd — COMMENTED** [review]
updated comments

File: docs/UserGuide.md (hunk 2 above)
Comment (dvdweien) [reply docs/UserGuide.md]: i tried my best to match the find format!

File: docs/UserGuide.md (hunk 3 above)
Comment (dvdweien) [reply docs/UserGuide.md]: I tried elaborating more, Is there a better way to phrase it? Was wondering if adding Option, Sales Agreement, Completion to the glossary would be good as well

File: docs/UserGuide.md [hunk 4]
```diff
@@ -215,16 +215,25 @@ Shows a list of all appointments in the app.
...
-* `proceed` or `cancel` can only be used on a property with an existing status
-* `proceed` would move the status on to the next one. e.g. Option to Sales Agreement or Sales Agreement to Completion
-* `cancel` would remove the status of the property
-* At least one of the optional fields must be provided.
+* Updates the status of the property at the specified `INDEX`. The index refers to the index number shown in the displayed list. The index **must be a positive integer** 1, 2, 3, …​
+  
+Options:
+* `u/new AMOUNT`
+  
+    The `new` keyword can only be used on a property without an existing status
```
Comment (w2vgd) [inline docs/UserGuide.md]: Maybe state what the amount means also?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -215,16 +215,25 @@ Shows a list of all appointments in the app.
...
+  
+
+* `u/proceed`
+  
+    Can only be used on a property with an existing status. `proceed` would move the status on to the next one. e.g. Option to Sales Agreement or Sales Agreement to Completion where Option represents the state when the Option to Purchase has been exercised for the property, Sales Agreement represents the state when the Sales and Purchase Agreement has been issued and Completion represents the state when the sales of the property has completed.
```
Comment (w2vgd) [inline docs/UserGuide.md]: Would prefer a nested table to explain what option, sales agreement and completion mean here. But ignore me if you don't think its better

File: docs/UserGuide.md [hunk 6]
```diff
@@ -377,8 +386,7 @@ Action | Format, Examples
 **List all** | `list all`
 **List property** | `list property`
 **List appointment** | `list appointment`
-**Add new status** | `update INDEX new AMOUNT`<br><br> e.g.,`update 1 new 600000`
-**Update status** | `update INDEX u/STATUS`<br><br> e.g. `update 3 u/proceed`
+**Update status** | `update INDEX OPTION`<br><br> Options: <br><ul> <li>`u/new AMOUNT`</li> <li>`u/proceed`</li> <li>`u/cancel`</li> </ul> e.g.,`update 1 u/new 600000`
```
Comment (w2vgd) [inline docs/UserGuide.md]: There will be error parsing the html tags here. Try following what i did for the `find property` command below if you can. If not I will fix it after this is merged ba

**Review by w2vgd — COMMENTED** [review]
Some extra suggestions

File: docs/UserGuide.md (hunk 4 above)
Comment (dvdweien) [reply docs/UserGuide.md]: added in elaboration!

File: docs/UserGuide.md (hunk 5 above)
Comment (dvdweien) [reply docs/UserGuide.md]: added in a table!

File: docs/UserGuide.md (hunk 6 above)
Comment (dvdweien) [reply docs/UserGuide.md]: i tried my best to follow it

File: docs/UserGuide.md (hunk 5 above)
Comment (w2vgd) [reply docs/UserGuide.md]: omg im so sorry i meant like a nested list like that

> - ...
> - ...

cuz a table would be kind of out of place right?

Comment (dvdweien) [reply docs/UserGuide.md]: changed the table to a list

**Review by w2vgd — COMMENTED** [review]
LGTM but do you want to put the class diagram as a separate PR since you are not using it in the UG yet?

**Comment by dvdweien** [comment]
> LGTM but do you want to put the class diagram as a separate PR since you are not using it in the UG yet?

oops, my bad. Removed it!

**Review by w2vgd — APPROVED** [review]
LGTM

**Pull request merged**